kerml/src/examples/Simple Tests/Filtering.kerml

```diff
@@ -18,18 +18,32 @@ package Filtering {
 	            level = 2;
 	        }
 	    }
+		composite feature system : System;
 	}
-	
+
 	package UpperLevelApprovals {
 	    import DesignModel::**;
-	    filter @Annotations::ApprovalAnnotation and 
-	           Annotations::ApprovalAnnotation::approved and 
+	    filter Annotations::ApprovalAnnotation::approved and 
 	           Annotations::ApprovalAnnotation::level > 1;
+	    
+	    struct Test :> System;
 	}
 	
 	package UpperLevelApprovals1 {
 	    import Annotations::**;
-	    import DesignModel::**[@ApprovalAnnotation and approved and level > 1];
+	    import DesignModel::**[@Structure][approved and level > 1];
```

Comment:
No, the "approved" and "level" names are resolved without qualification because of the recursive "import Annotations::**;". While this model is more or less based on an early example, it is really is just a test if various things related to filtering work. The real examples on this are documented in the training material.